package.json

```diff
@@ -18,31 +18,29 @@
     "cozydb": "0.1.10",
     "graceful-fs": "3.0.8",
     "http-proxy": "1.12.0",
-    "jade": "1.11.0",
     "locale": "0.0.14",
     "lockedpath": "0.1.1",
     "node-polyglot": "0.3.0",
     "passport": "0.2.0",
     "passport-local": "1.0.0",
     "printit": "0.1.15",
+    "pug-runtime": "2.0.0",
```

Comment:
We are using `jade 1` below and `pug-runtime 2` here. It's not the same major version. Is it OK to do so?